modules/dockermodelrunner/internal/sdk/client/pull.go

```diff
@@ -18,19 +20,48 @@ func (c *Client) PullModel(ctx context.Context, fullyQualifiedModelName string)
 		return fmt.Errorf("new post request (%s): %w", reqURL, err)
 	}
 
-	log.Default().Printf("🙏 Pulling model %s. Please be patient, no progress bar yet!", fullyQualifiedModelName)
+	log.Default().Printf("🙏 Pulling model %s. Please be patient", fullyQualifiedModelName)
```

Comment:
suggestion: create a logger var so we don't have to run the Default function multiple times, see example in the for loop suggestion below for an example.